src/Twig/Extension/ArrayExtension.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Bolt\Twig\Extension;
+
+use Bolt\Twig\Runtime;
+use Twig_Extension as Extension;
+
+/**
+ * Array functionality Twig extension.
+ *
+ * @internal
+ *
+ * @author Gawain Lynch <gawain.lynch@gmail.com>
+ */
+class ArrayExtension extends Extension
```

Comment:
Yeah, other thing I was wanting to ping you on, but family commitments got you yesterday. Fabien has everything [under one namespace in `symfony/twig-bridge`](https://github.com/symfony/twig-bridge/tree/3.2/Extension) and wondering if we shouldn't head in that direction too?

The problem I had was all but one runtime/definition separate, and that being confusing. But so very open to discussion on this.